bin/sparkling-env.sh

```diff
@@ -118,11 +118,19 @@ if [ -z "$TOPDIR" ]; then
   exit -1
 fi
 
-function checkJava(){
+function checkJava() {
 if [ -z $(which java) ]; then
     echo "Java is not installed. Please install Java first before continuing with Sparkling Water."
     exit -1
 fi
+
+if [ "$SPARK_MAJOR_VERSION" != "2.1" ]; then
```

Comment:
Should there be a check for Spark 2.1 -> java 7+

Reply:
Good point, will do